google/cloud/pubsub/samples/samples.cc

```diff
@@ -2125,7 +2126,8 @@ void AutoRunAvro(
   std::cout << "\nRunning DeleteSchema() sample [avro]" << std::endl;
   DeleteSchema(schema_admin, {project_id, avro_schema_id});
 
-  CleanupSchemas(schema_admin, project_id, absl::Now());
+  std::chrono::system_clock clock;
+  CleanupSchemas(schema_admin, project_id, absl::FromChrono(clock.now()));
```

Comment:
nit-ish: this is a static function, you can say `std::chrono::system_clock::now()`